tests/manage/mcg/test_host_node_failure.py

```diff
@@ -203,4 +203,5 @@ def test_noobaa_sts_host_node_failure(
         self.sanity_helpers.health_check()
 
         # Creates bucket then writes, reads and deletes objects
-        self.sanity_helpers.obc_put_obj_create_delete(mcg_obj, bucket_factory)
+        # TODO: Reduce timeout in future versions once 2028559 is fixed
+        self.sanity_helpers.obc_put_obj_create_delete(mcg_obj, bucket_factory, 900)
```

Comment:
I would suggest to call it via `timeout=900` instead of just `900` to make it more clear and more resilient to any future changes.